pom.xml

```diff
@@ -730,10 +730,11 @@
         <artifactId>guava</artifactId>
         <version>18.0</version>
       </dependency>
+      <!-- https://mvnrepository.com/artifact/org.mnode.ical4j/ical4j -->
       <dependency>
-        <groupId>net.fortuna.ical4j</groupId>
-        <artifactId>ical4j</artifactId>
-        <version>1.0-rc2-patched</version>
+       <groupId>org.mnode.ical4j</groupId>
+       <artifactId>ical4j</artifactId>
+       <version>2.2.0</version>
```

Comment:
@karendolan Is there a specific reason to not use version 3.0.2 of ical4j?

Reply:
Hi @staubesv, there was no reason. I tested with the newer 3.0.2  and had no issues so I've updated the pull with that version. Thank you for seeing this.